### ISSUE TITLE
Pass "never" to scrollviewport to hide scrollbar

### ISSUE THREAD

**Issue by enesusta** [icflorescu/mantine-datatable#327]
As you probably know, if we pass type="never" to mantine scrollarea then scrollbar will disappear

```tsx
import { ScrollArea } from '@mantine/core';

function Demo() {
  return (
    <ScrollArea h={250} type="never" scrollHideDelay={6000}>
      {/* ... content */}
    </ScrollArea>
  );
}
```

I'd be great if we can pass mantine scrollarea objects to datatable.

```tsx
const scrollViewportProps = {
  type: "never",
  ...others
}


<DataTable
            scrollViewportProps={scrollViewportProps}
           />
```

**Comment by enesusta** [comment]
done with #331

**Comment by icflorescu** [comment]
Fixed in `v2.6.0` thanks to @enesusta 🙏
